source/cellular_bg96_api.c

```diff
@@ -42,7 +42,7 @@
 
 /*-----------------------------------------------------------*/
 
-#define CELLULAR_AT_CMD_TYPICAL_MAX_SIZE           ( 32U )
+#define CELLULAR_AT_CMD_TYPICAL_MAX_SIZE           ( 64U )

```

Comment:
Is this truly necessary? Can we create a more specific define to be used in the particular set of commands that are larger? I'm asking because this will have impact in the memory footprint

Reply:
Yes, we can this was needed for the AT+QSSLCFG command if I'm not mistaken.

Reply:
Reverted and added define for SSLCFG